src/Razor/test/Microsoft.VisualStudio.LanguageServices.Razor.Test/Microsoft.VisualStudio.LanguageServices.Razor.Test.csproj

```diff
@@ -31,6 +31,16 @@
     <EmbeddedResource Include="TestFiles\**\*" />
   </ItemGroup>
 
+  <!--<ItemGroup>
+    <Reference Include="C:\Projects\roslyn\artifacts\bin\Microsoft.CodeAnalysis\Debug\netstandard2.0\Microsoft.CodeAnalysis.dll" />
+    <Reference Include="C:\Projects\roslyn\artifacts\bin\Microsoft.CodeAnalysis.CSharp\Debug\netstandard2.0\Microsoft.CodeAnalysis.CSharp.dll" />
+    <Reference Include="C:\Projects\roslyn\artifacts\bin\Microsoft.CodeAnalysis.ExternalAccess.Razor\Debug\netstandard2.0\Microsoft.CodeAnalysis.ExternalAccess.Razor.dll" />
+    <Reference Include="C:\Projects\roslyn\artifacts\bin\Microsoft.CodeAnalysis.Workspaces\Debug\netstandard2.0\Microsoft.CodeAnalysis.Workspaces.dll" />
+    <Reference Include="C:\Projects\roslyn\artifacts\bin\Microsoft.CodeAnalysis.EditorFeatures.Text\Debug\netstandard2.0\Microsoft.CodeAnalysis.EditorFeatures.Text.dll" />
+    <Reference Include="C:\Projects\roslyn\artifacts\bin\Microsoft.CodeAnalysis.CSharp.Features\Debug\netstandard2.0\Microsoft.CodeAnalysis.CSharp.Features.dll" />
+    <Reference Include="C:\Projects\roslyn\artifacts\bin\Microsoft.CodeAnalysis.LanguageServer.Protocol\Debug\netstandard2.0\Microsoft.CodeAnalysis.LanguageServer.Protocol.dll" />
+  </ItemGroup>-->
```

Comment:
What were these all used for? Just your testing of local Roslyn bits?

Reply:
Oh lol, good catch. Yes, it was just me at various points using locally built Roslyn bits when they weren't publicly available yet.